### PR TITLE
Workspace Extension Functionality

### PR DESCRIPTION
**Pull request by vkmb**
By default, Zethus uses the `ROS_PACKAGE_PATH` set before Jupyter Hub launches. 

To use a new ROS workspace inside of Jupyter Lab, the existing procedure is,
1. Stop the jupyter lab instance.
1. Source the new workspace.
1. Start the jupyter lab instance.

This workflow is doable in a local instance. The problem arises with the Jupyter Hub instance launched from a Dockerfile. 

In the Jupyter Hub instance with the current Jupyterlab-ros server plugin, it's impossible to add the new workspaces and fetch new robot models making it cumbersome to visualise robot models. 

The updated plugin tries to solve this problem by enabling the user to add and delete workspaces as required for the task without restarting the Jupyter lab, copying the packages to the base path, modifying the Dockerfile and restarting Jupyter Hub instance.
To test the plugin
```
docker run --rm -p 8888:8888 kumkum/jupyterhub:jupyterlab-ros-stable
```
- The updated plugin can be accessed with the [following docker image](https://hub.docker.com/layers/kumkum/jupyterhub/jupyterlab-ros-stable/images/sha256-1df20e8deba0915c692393b496717ffbb66c4c10f17c64503b69090fa37559cc?context=repo).
- Implements #7

### CODE REVIEW

File: js/schema/settings.json [hunk 1]
```diff
@@ -14,6 +14,12 @@
       "title": "Master launch file",
       "description": "Absolute path of your launch file to initialize ROS master.\nKeep it empty to use the default one.",
       "default": ""
+    },
+    "workspaces": {
+      "type": "string",
+      "title": "Workspace Paths",
+      "description": "Absolute paths to the workspaces separated by colons.\nExample:\n/home/jovyan/ws1:/home/jovyan/w2:/home/jovyan/ws3\nKeep it empty to use load the workspaces sourced before jupyter lab was started.",
```
Comment (ihuicatl) [inline js/schema/settings.json:21]: Tiny typo in "/w**s**2"

File: jupyter_ros_server/api/pkgs.py [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import os
 import json
+from turtle import st
```
Comment (ihuicatl) [inline jupyter_ros_server/api/pkgs.py:3]: It doesn't seem like you need this.

File: jupyter_ros_server/static/settings.json [hunk 3]
```diff
@@ -1 +1 @@
-{"env": "", "master": ""}
\ No newline at end of file
+{"env": "", "master": "", "workspaces": "/Users/kunkuma/ros_x_mac/dev_ws:/Users/kunkuma/ros_x_mac/igor_ws"}
```
Comment (ihuicatl) [inline jupyter_ros_server/static/settings.json:1]: It would be better to default to an empty string here.

**Review by ihuicatl — CHANGES_REQUESTED** [review]
Hi @vkmb ,

thanks for your PR! Looks great, just needs some minor changes :grin:

File: js/schema/settings.json (hunk 1 above)
Comment (vkmb) [reply js/schema/settings.json]: Thanks catching it, I had fixed it now.

File: jupyter_ros_server/api/pkgs.py (hunk 2 above)
Comment (vkmb) [reply jupyter_ros_server/api/pkgs.py]: I don't know when I added this one. 😅

File: jupyter_ros_server/static/settings.json (hunk 3 above)
Comment (vkmb) [reply jupyter_ros_server/static/settings.json]: Thanks for mentioning it, removed it in the latest commit.

**Comment by hbcarlos** [comment]
This is awesome. Thanks for working on this PR @vkmb!

**Review by ihuicatl — APPROVED** [review]
Excellent!

**Pull request merged**